### PR TITLE
Support overwriting the gcpt restorer manually

### PR DESCRIPTION
**Pull request by xyyy1420**

### CODE REVIEW

File: src/test/csrc/verilator/emu.cpp
Comment (shinezyy) [inline src/test/csrc/verilator/emu.cpp:1]: overwrite_nbytes` should not use 'o' as shorthand

**Review by shinezyy — CHANGES_REQUESTED** [review]
overwrite_nbytes should not use 'o' as shorthand, because o is usually used for output or some common options.

please remove 'o' from L161, L172, and L263

**Review by shinezyy — DISMISSED** [review]
seems Ok

File: src/test/csrc/verilator/emu.cpp [hunk 1]
```diff
@@ -83,6 +84,7 @@ static inline void print_help(const char *file) {
   printf("  -R, --ipc-interval=NUM     the interval insts of drawing IPC curve\n");
 #endif
   printf("  -X, --fork-interval=NUM    LightSSS snapshot interval (in seconds)\n");
+  printf("      --overwrite-nbytes=n   set valid bytes,but less than 0xf00,default value is 0x700\n");
```
Comment (poemonsense) [inline src/test/csrc/verilator/emu.cpp:87]: The default value here is different from the value in `uint64_t overwrite_nbytes = 0xe00;`? 

Better to use `default: 0xe00` to reduce the length of the message and add a whitespace after `,`

**Review by poemonsense — DISMISSED** [review]

File: src/test/csrc/verilator/emu.cpp [hunk 2]
```diff
@@ -368,6 +374,12 @@ Emulator::Emulator(int argc, const char *argv[]):
     }
   }
 
+  if (args.gcpt_restore) {
+    InputReader *reader=new FileReader(args.gcpt_restore);
```
[the remark this answers is not in the document]
Comment (poemonsense) [reply src/test/csrc/verilator/emu.cpp:378]: and a whitespace after `,`

**Pull request merged**